### PR TITLE
jmc page added

### PR DESCRIPTION
**Pull request by hendrikebbers**
Page for Java Mission Control added
jmc

##### Checklist

- [x ] `npm test` passes
- [ x] documentation is changed or added (if applicable)
- [ x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by hendrikebbers** [comment]
Fix for https://github.com/AdoptOpenJDK/openjdk-website/issues/582

**Comment by hendrikebbers** [comment]
@thegreystone @reinhapa can you have a look at the PR?

**Comment by aahlenst** [comment]
Linking to ci.adoptopenjdk.net for downloads does not feel right. Is this only an interim solution?

**Comment by jerboaa** [comment]
Staging URL is: https://staging.adoptopenjdk.net/583/jmc.html

**Comment by reinhapa** [comment]
@aahlenst I think we got two different parts to solve here AFAIK:
Firstly Having a introductory page as done by other providers in order to point to our builds of JMC, secondly providing a enhanced update site informations given within the update repositories `index.html`, that will not served as `applicaiton/html` content type by the Artifactory unfortunately...

**Comment by ggam** [comment]
Isn't it JDK Mission Control Now? Java Mission Control refers to the Oracle product I think.

**Comment by thegreystone** [comment]
> Linking to ci.adoptopenjdk.net for downloads does not feel right. Is this only an interim solution?

Agreed. It would be nicer to properly host them. Also, the update sites need to be unpacked and hosted somewhere, like discussed in #582, so that the update sites can be used directly by the RCP application build.

**Comment by hendrikebbers** [comment]
I changed the name to "JDK Mission Control"

About the link to ci.adoptopenjdk.net: No idea what to do here. Maybe @karianna or @gdams can add some info?

**Comment by aahlenst** [comment]
I had some discussions with @reinhapa regarding hosting. Some things are currently on Artifactory (https://adoptopenjdk.jfrog.io/). But I wasn't able to figure out if anything is wrong with that or needs improvement.

**Comment by thegreystone** [comment]
I think it would be nice to unpack the update sites somewhere. See the first part of #582.

**Comment by thegreystone** [comment]
> I think it would be nice to unpack the update sites somewhere. See the first part of #582.

To elaborate a bit on that - unless they are unpacked somewhere and served up, they cannot be referenced from the RCP application.

**Comment by reinhapa** [comment]
> I think it would be nice to unpack the update sites somewhere. See the first part of #582.
@thegreystone  The update site is already available unpacked under https://adoptopenjdk.jfrog.io/adoptopenjdk/p2-jmc-snapshots/ide
and
https://adoptopenjdk.jfrog.io/adoptopenjdk/p2-jmc-snapshots/rcp
the only problem, that I see there is that the `index.html` contains links to other sites, that do no longer exist or are not relative to the current location. I would suggest to publish an alternative `ìndex.html` that is done in some AdoptOpenJDK style using also AdoptOpenJDK locations...

**Comment by thegreystone** [comment]
Ok! Excellent! Well, in that case, overriding the index.html files seems to be the last remaining thing to do! :) Let me know if I can do anything to help!

**Comment by thegreystone** [comment]
@reinhapa - https://adoptopenjdk.jfrog.io/adoptopenjdk/p2-jmc-snapshots/ide it that the latest snapshot build? Are the 7.0.0 release builds available too?

**Comment by reinhapa** [comment]
> @reinhapa - https://adoptopenjdk.jfrog.io/adoptopenjdk/p2-jmc-snapshots/ide is that the latest snapshot build? Are the 7.0.0 release builds available too?

@thegreystone at the moment not, but that should be possible. I will have to check the URL's to use got the final versions..

**Comment by thegreystone** [comment]
Ok cool!

I noticed that some of the files that should be available in the built update site isn't. For example, I can't find https://adoptopenjdk.jfrog.io/adoptopenjdk/p2-jmc-snapshots/ide/images/bullet-arrow-red.gif.

**Comment by thegreystone** [comment]
Hi all! Anything I can do to help conclude this? :)

File: src/handlebars/jmc.handlebars [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{{> header title='JDK Mission Control | ' }}
+
+<main class="grey-bg">
+
+  <h1 class="margin-bottom large-title">JDK Mission Control</h1>
+<div class="info-page-container upstream blue-link-section">
+  <div class="download">
+    <p>JDK Mission Control is an open source tools suite for the JVM. It help to find problems in programs and optimize programs at runtime.</p>
```
Comment (jerboaa) [inline src/handlebars/jmc.handlebars]: `It helps to ... ` over `It help to`.

Comment (karianna) [reply src/handlebars/jmc.handlebars]: Fixed inline

**Pull request merged**